fundingmanager.go

```diff
@@ -2756,10 +2755,11 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 		ourDustLimit = defaultLitecoinDustLimit
 	}
 
-	fndgLog.Infof("Initiating fundingRequest(localAmt=%v, remoteAmt=%v, "+
-		"capacity=%v, chainhash=%v, peer=%x, dustLimit=%v, min_confs=%v)",
-		localAmt, msg.pushAmt, capacity, msg.chainHash,
-		peerKey.SerializeCompressed(), ourDustLimit, msg.minConfs)
+	fndgLog.Infof("Initiating fundingRequest(localAmt=%v "+
+		"(subtractFees=%v), pushAmt=%v, chainhash=%v, peer=%x, "+
+		"dustLimit=%v, min_confs=%v)", localAmt, msg.subtractFees,
```

Comment:
Style nit: make `min_confs` camelCase for consistency?

Reply:
We kinda use both in the codebase for logging atm. Personally, I prefer snake case for stuff like this, and camel case when referring to a sub-system within the codebase, similar to Python naming conventions.

Reply:
Made them all snake case 🐍